app/Models/User.php

```diff
@@ -16,11 +16,17 @@ class User extends LaravelUser
         "email",
         "password",
     ];
+    protected $quarded = [
```

Comment:
I think it should be `guarded` with G.

Reply:
It's typo. PHPStorm didn't warn me. Strange